components/CheckDocumentation/index.tsx

```diff
@@ -36,6 +37,7 @@ const Docs: {
   },
   index_advisor: {
     missing_index: MissingIndex,
+    index_insights: IndexInsights,
```

Comment:
Note that internally this is named `IndexInsights` / `index_insights`, since future work will enable insights that recommend dropping indexes. Since changing some of these internal names is harder, we want to use a more generic name there right off the bat.